attoparsec-iso8601/attoparsec-iso8601.cabal

```diff
@@ -40,7 +40,7 @@ library
     base >= 4.9 && < 5,
     base-compat-batteries >= 0.10.0 && < 0.13,
     time-compat >= 1.9.4 && < 1.10,
-    text >= 1.2.3.0 && < 1.3.0.0,
+    text >= 1.2.3.0 && < 1.3.0.0 || >= 2.0 && <2.1,
```

Comment:
Why not `>= 1.2.3.0 && < 2.1`?

Reply:
I don't think there will be any `text-1.3` releases, but just to be on the safe side.